lua/lualine/components/signify.lua

```diff
@@ -1,33 +1,89 @@
--- Copyright (c) 2020-2021 hoob3rt
+-- Copyright (c) 2020-2021 shadmansaleh
 -- MIT license, see LICENSE for more details.
 
-local function signify()
-   if vim.fn.exists('*sy#repo#get_stats') == 0 then return '' end
-   local added, modified, removed = unpack(vim.fn['sy#repo#get_stats']())
-   if added == -1 then return '' end
-   local symbols = {
-     '+',
-     '-',
-     '~',
-   }
-   local result = {}
-   local data = {
-    added,
-    removed,
-    modified,
-   }
-   for range=1,3 do
-     if data[range] ~= nil and data[range] > 0
-       then table.insert(result,symbols[range]..''..data[range]..' ')
-     end
-   end
-
-   if result[1] ~= nil then
-       return table.concat(result, '')
-   else
-       return ''
-   end
+local utils = require'lualine.utils.utils'
+
+local default_color_added    = "#f0e130"
+local default_color_removed  = "#90ee90"
+local default_color_modified = "#ff0038"
+
+local function get_highlight(scope, color_group)
+  local color = string.format('#%06x', vim.api.nvim_get_hl_by_name(color_group, true)[scope])
```

Comment:
ok lol, next time I will defo search for vim.api functions instead of implementing a mess myself :laughing: